src/compas_view2/objects/__init__.py

```diff
@@ -33,3 +39,8 @@
 Object.register(Sphere, SphereObject)
 Object.register(Torus, TorusObject)
 Object.register(Arrow, ArrowObject)
+
+try:
+    Object.register(Block, MeshObject)
+except:
+    pass
```

Comment:
```python
try:
    from compas_assembly.datastructures import Block
except ModuleNotFoundError:
    pass
else:
    Object.register(Block, MeshObject)
```

Reply:
@brgcode changed in ef3b803b5c8d8b6347b1532a6d5db78dde8f19ee